### PR TITLE
Pv update gkl utils

### LINKED ISSUE #33: Error checking for AVX support
We have received reports of GKL hanging while checking for AVX support.

It appears the `grep -i avx /proc/cpuinfo` process hangs forever in Java. However, the `grep` command runs fine under bash.

We should implement a more robust AVX check that does not rely on `grep` and works on multiple platforms.

### PR DESCRIPTION
**Pull request by pnvaidya**
Resolves #33.
Modified GKLUtils (removed previous implementation of checkAVXfeature). Removed FTZ as separate library and added functions as part of GKLUtils

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <ConfirmationsSetting value="0" id="Remove" />
   </component>
   <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/build/classes" />
+    <output url="file://$PROJECT_DIR$/classes" />
```
Comment (gspowley) [inline .idea/misc.xml]: Please remove this file from the commit.

File: profile.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+level, time(sec), filesize
```
Comment (gspowley) [inline profile.csv]: Please remove this file from the commit.

File: src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+public class IntelGKLUtilsUnitTest {
+    private final static Logger log = LogManager.getLogger(IntelGKLUtilsUnitTest.class);
+
+    @Test(enabled = true)
+    public void simpleTest() {
+
+        IntelGKLUtils utils = new IntelGKLUtils();
+
+        boolean isLoaded = utils.load(null);
+
```
Comment (gspowley) [inline src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java]: Add `assert(isLoaded);`

File: src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    @Test(enabled = true)
+    public void childThreadTest() {
+
+        log.info("Parent setting FTZ = true");
+
+        IntelGKLUtils utils = new IntelGKLUtils();
+
+        boolean isLoaded = utils.load(null);
+
```
Comment (gspowley) [inline src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java]: Add `assert(isLoaded);`

File: src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    @Test(enabled = true)
+    public void platformFeaturesTest() {
+
+        log.info("Check Platform Features");
+
+        IntelGKLUtils utils = new IntelGKLUtils();
+
+        boolean isLoaded = utils.load(null);
+
```
Comment (gspowley) [inline src/test/java/com/intel/gkl/IntelGKLUtilsUnitTest.java]: Add `assert(isLoaded);`

**Comment by pnvaidya** [comment]
Modified GKLUtils (removed previous implementation of checkAVXfeature). Removed FTZ as separate library and added functions as part of GKLUtils.

**Pull request merged**